apps/admin/frontend/src/components/smartcard_modal/card_details_view.tsx

```diff
@@ -166,9 +166,7 @@ export function CardDetailsView({
   return (
     <React.Fragment>
       {actionStatus?.status === 'Error' && (
-        <P align="center">
-          <SuccessOrErrorStatusMessage actionStatus={actionStatus} />
-        </P>
+        <SuccessOrErrorStatusMessage actionStatus={actionStatus} />
```

Comment:
This `P` is unnecessary because `SuccessOrErrorStatusMessage` always returns a `P` and the modal centers the content.